internal/pkg/cli/deploy/workload.go

```diff
@@ -56,6 +62,14 @@ const (
 	labelForVersion       = "com.aws.copilot.image.version"
 	labelForContainerName = "com.aws.copilot.image.container.name"
 )
+const (
+	paddingForBuildAndPush      = 5
```

Comment:
nit: I'm trying to think how we can hint that this is number of spaces, maybe this?
```suggestion
	paddingInSpacesForBuildAndPush      = 5
```